lib/pages/event_creation/event_creation.dart

```diff
@@ -0,0 +1,135 @@
+import 'package:firebase_core/firebase_core.dart';
+import 'package:firebase_database/firebase_database.dart';
+import 'package:flutter/material.dart';
+import 'package:regatta_buddy/pages/event_creation/event_form.dart';
+import 'package:regatta_buddy/pages/event_creation/event_route.dart';
+import 'package:regatta_buddy/pages/event_creation/event_social.dart';
+import 'package:regatta_buddy/pages/home.dart';
+import 'package:regatta_buddy/widgets/app_header.dart';
+import 'package:regatta_buddy/widgets/complex_marker.dart';
+import 'package:uuid/uuid.dart';
+
+class EventCreationPage extends StatefulWidget {
+  static const String route = '/event_creation';
+
+  const EventCreationPage({super.key});
+
+  @override
+  State<EventCreationPage> createState() => _EventCreationPageState();
+}
+
+class _EventCreationPageState extends State<EventCreationPage> {
+  DatabaseReference databaseReference = FirebaseDatabase.instanceFor(
+          app: Firebase.app(),
+          databaseURL:
+              " https://regattabuddy-default-rtdb.europe-west1.firebasedatabase.app")
```

Comment:
I would extract it to another file as we did with tile provider url 
https://github.com/SomeBuddies/RegattaBuddy/blob/master/lib/utils/externalAPIConstants.dart
Mb it is possible to have nice config file like in yaml format. Just a thought.

Reply:
There is a file in app directory google.json and it should behave the way, that you don't need to specify what instance you want but this didn't want to work for me